src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck started its work");
+    }
+
+    public void stopWork() {
+        System.out.println("Truck stopped working");

```

Comment:
The message in the `stopWork` method is informative, but ensure it clearly indicates the type of `Machine` as required by the checklist.